src/util/EditionCardUtils.tsx

```diff
@@ -220,4 +221,39 @@ export default class EditionCardUtils {
         : undefined;
     return universityPress !== undefined;
   }
+
+  static createGetContent = (nyplIdentityCookie: any, router: NextRouter) => {
+    const fetchWithAuth = async (fulfillUrl: string, proxyUrl?: string) => {
```

Comment:
Looks like this code could be reused to call other endpoints that need authentication besides the fulfill endpoint?

I think it's fine to keep it like this for now but we could consider abstracting this out to use with other endpoints later on.